addons/mail/models/mail_message.py

```diff
@@ -798,12 +798,11 @@ def set_message_done(self):
     @api.model
     def unstar_all(self):
         """ Unstar messages for the current partner. """
-        partner_id = self.env.user.partner_id.id
+        partner = self.env.user.partner_id
```

Comment:
up to you but I feel this extra variable is not really necessary

Reply:
Since I use partner in several lines, I think it is cleaner to define it as a variable.